app/lib/differential_expression_service.rb

```diff
@@ -1,9 +1,14 @@
 # handle launching differential expression ingest jobs
 class DifferentialExpressionService
   extend Loggable
-
-  # regex for matching annotation names for possible cell type analogs
+  # possible cell type analogs
   CELL_TYPE_MATCHER = /cell.*type/i
+  # possible clustering algorithm results
+  CLUSTERING_MATCHER = /(clust|seurat|leiden|louvain|_snn_res)/i
```

Comment:
In browsing the breadth of clustering names in the prod database, there was a set of clusterings named SNN_RES_0.XX (where XX was a series of different numbers). I realized that the leading underscore for snn_res would unnecessarily exclude that set of clusterings. Do you have any reservations about removing the underscore for that part of the CLUSTERING_MATCHER?

Reply:
Nope - I defer to you on that.

Reply:
[2955df0](https://github.com/broadinstitute/single_cell_portal_core/pull/2029/commits/2955df088726df4d617406a3378b810d53ec6377)